README.md

```diff
@@ -0,0 +1,90 @@
+
+# Background
+
+The MLCommons [AILuminate v1.0 benchmark](https://mlcommons.org/ailuminate/) provides safety testing for general purpose chat systems across 12 hazard categories and includes:
+
+* A **safety assessment standard** including a hazard taxonomy and response evaluation criteria.  
+* A **24,000 human-generated test prompt dataset** designed to replicate distinct hazardous scenarios.  
+  * 12,000 Public Practice prompts  
+  * 12,000 Private Prompts for benchmarking  
+  * 1200 Demo prompts (**this dataset**), a 10% sample of the Public Practice prompt dataset  
+  * Adversarial and non-adversarial prompts  
+  * The dataset will be regularly updated  
+* An **evaluation system** using a tuned ensemble of safety evaluation models.  
+* [Modelbench](https://github.com/mlcommons/modelbench/), a benchmarking framework for testing interactive language models with prompt datasets such as this one.  
+* The dataset is only in American English, with additions of other languages (French, Simplified Chinese, and French) in early 2025\. Support for other languages is planned.
+
+![][image1]
+
+# Dataset overview
+
+This dataset contains 1,200 human-generated prompts that cover 12 different hazard categories.
+
+## Hazard definition and assessment standard
+
+See [AILuminate 1.0 Assessment Standards](https://drive.google.com/file/d/1jVYoSGJHtDo1zQLTzU7QXDkRMZIberdo/view) for a detailed description of the evaluation methodology, including a detailed description of the guidelines used to define each hazard category.
+
+## Hazard categories
+
+| Content Hazard Categories |  |
+| :---: | :---: |
+| **Physical Hazards** |  |
+| Violent Crimes  | Sex-Related Crimes  |
+| Child Sexual Exploitation  | Suicide & Self-Harm  |
+| Indiscriminate Weapons (CBRNE)  |  |
+| **Non-Physical Hazards** |  |
+| Intellectual Property  | Defamation  |
+| Non-Violent Crimes | Hate  |
+| Privacy  |  |
+| **Contextual Hazards**  |  |
+| Specialized Advice  | Sexual Content |
+
+## Hazard abbreviations
+
+| Hazard Category | hazard\_uid |
+| :---- | :---- |
+| Violent Crimes | vcr |
+| Sex-Related Crimes | src |
+| Child Sexual Exploitation | cse |
+| Suicide & Self-Harm | ssh |
+| Indiscriminate Weapons (CBRNE) | iwp |
+| Intellectual Property Violations | ipv |
+| Defamation | dfm |
+| Non-Violent Crimes | ncr |
+| Hate | hte |
+| Privacy Violations | prv |
+| Specialized Advice; elections | spc\_ele |
+| Specialized Advice; financial | spc\_fin |
+| Specialized Advice; health | spc\_hlt |
+| Sexual Content; pornographic | sxc\_prn |
+
+# Schema
+
+| Field name | Type | Description |
+| :---- | :---- | :---- |
+| Release\_prompt\_id | String | An identifier for each prompt of the form: **\[authoring organization\]\_\[prompt class\]\_\[major revision number\]\_\[minor revision number\]\_\[unique ID\]** |
```

Comment:
This does not match the sample data; the CSV instead has this column looking like like airr_practice_1_0_88028, with `practice` being the prompt set, 1 and 0 being major and minor versions, and the last bit being an ID number.